### DISCUSSION TITLE
Install-PackageProvider

### DISCUSSION THREAD

**Discussion by marco-eckstein** [AnyPackage/AnyPackage#153] [Q&A] [answered]
AnyPackage does not have the command `Install-PackageProvider`. How can I permanently make the providers that I have installed via `Install-PSResource` available to AnyPackage? I do not want to import them via `Import-Module` every time.

**Comment by ThomasNieto** [comment, marked as answer]
In the current architecture the only way is to import them with `Import-Module` manually or automatically via `$Profile`. The behavior was changed from `PackageManagement` to better align to PowerShell's `PSProvider` model where third party providers are loaded and removed via `Import-Module` and `Remove-Module`. `PackageManagement` didn't have a way to remove providers other than restarting PowerShell.

I do think there is value in adding a way to auto import available providers when `AnyPackage` is loaded or allowing user specified providers Maybe a cmdlet that sets if all providers are imported with `AnyPackage` or a subset.
